content/developer-guide/data-views-dev/get-data-view-data/quick-start-get-data-view-data.md

```diff
@@ -115,9 +115,7 @@ The view data is available in several formats. A format can be specified using t
 
 #### Response format
 
-By default, the value of the header `Accept-Verbosity` is `verbose` for the response format. Verbose responses include all values in the returned JSON payload, including defaults. When set to `non-verbose`, properties with null values are omitted from the response to reduce bandwidth. This behavior is only applicable to the JSON format, as all other formats represent a table which cannot have omitted values.
-
-Note that the `Accept-Verbosity` header for SDS differs as its default is set to `non-verbose`.
+By default, the value of the header `Accept-Verbosity` is `verbose` for the response format. For more information on response formats, see [AVEVA Data Hub API reference](xref:osisoftCloudServices).
```

Comment:
Update this lead-in text to match the lead in text for the next topic. This content is weird--it's talking about options that aren't relevant until you set a response format